pxfish/field_type.py

```diff
@@ -197,42 +240,53 @@ def types_valid(*, operation_type, definitions, force, session):
         force (Boolean): if set, overrides conflict checks
         session (Session Object): Aquarium session object
     """
-    field_types = session.FieldType.where({'parent_id': operation_type.id})
+    aquarium_field_types = session.FieldType.where({'parent_id': operation_type.id})
 
     missing_inputs, input_conflicts, valid_inputs = check_for_conflicts(
-        field_types=[t for t in field_types if t.role == 'input'],
+        aquarium_field_types=[t for t in aquarium_field_types if t.role == 'input'],
         definitions=definitions['inputs'],
```

Comment:
These list comprehensions should be put into methods, e.g., `inputs(field_types)`